libs/designsystem/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@kirbydesign/designsystem",
-  "version": "10.5.1",
+  "version": "10.5.3",
```

Im guessing leftovers from testing? 😄 
```suggestion
  "version": "10.5.1",
```